script/client

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Tries to start a browser, if an opensearch server is currently running using docker compose.
```

Comment:
Im asuming this is trying to open openagency example client or ? 
In that case, the following is not correct : 

Tries to start a browser, if an opensearch server is currently running using docker compose.